pkg/api/const.go

```diff
@@ -485,6 +485,12 @@ const (
 	DefaultWindowsSSHEnabled = true
 )
 
+// WindowsProfile defaults
+// TODO: Move other values defined in WindowsProfiles (like DefaultWindowsSSHEnabled) here.
+const (
+	DefaultWindowsProvisioningScriptsPackageURL = "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.1.zip"
```

Comment:
Should we add this to [DefaultKubernetesSpecConfig  in azenvtypes.go](https://github.com/Azure/aks-engine/blob/ea9e31baf3009c6bed621b72a4607611d3b7c361/pkg/api/azenvtypes.go#L88)?
We need to assure that the new package is ready for AzureChinaCloud. AzureChinaCloud should use `https://mirror.azk8s.cn/aks-engine`.
I think that we should have a rule to manage so many folders for different packages so we do not need to reconfigure  the proxy for AzureChinaCloud.
FYI @andyzhangx

Reply:
@jchauncey would you also sync this `https://kubernetesartifacts.azureedge.net/aks-engine` to `https://kubernetesartifacts.core.chinacloudapi.cn/aks-engine`, every bits in `https://kubernetesartifacts.azureedge.net` should sync to `https://kubernetesartifacts.core.chinacloudapi.cn` otherwise it may break sth on Azure China.

Reply:
@abelhu @andyzhangx I see if the Azure DevOps release pipeline that this package was uploaded to https://kubernetesartifacts.blob.core.chinacloudapi.cn/aks-engine/windows/provisioning/signedscripts-v0.0.1.zip.

@AbelHu are you saying that we should add an entry to KubernetesSpecConfig that points to https://kubernetesartifacts.azureedge.net/aks-engine by default?
I'm not familiar with how other URLs for AzureChinaCloud are managed by aks-engine. 
If you have an example to follow I would appreciate it.

Also @jackfrancis as FYI here

Reply:
I think that we need to add a new property `WindowsProvisioningScriptsPackageURL` in `KubernetesSpecConfig`, set DefaultKubernetesSpecConfig.WindowsProvisioningScriptsPackageURL to "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-" + SignedWindowsScriptsVer + ".zip" for global Azure and expect to set AzureChinaCloudSpec.KubernetesSpecConfig.WindowsProvisioningScriptsPackageURL to "https://mirror.azk8s.cn/aks-engine/windows/provisioning/signedscripts-" + SignedWindowsScriptsVer + ".zip".

Reply:
Thanks @AbelHu. Let me add that.

Reply:
@AbelHu @andyzhangx  please take a look at the last commit in the PR where I added WindowsProvisioningScriptsPackageURL  to kubernetesSpecConfig